app/controllers/api/teams.php

```diff
@@ -502,6 +502,7 @@
                     ->setFrom($from)
                     ->setRecipient($invitee->getAttribute('email'))
                     ->setName($invitee->getAttribute('name'))
+                    ->setSmtp($project->getAttribute('smtp', []))
```

Comment:
Same suggestions about the complex structure.

Reply:
updated